content/en/docs/concepts/configuration/_index.md

```diff
@@ -2,6 +2,134 @@
 title: "Configuration"
 weight: 80
 description: >
-  Resources that Kubernetes provides for configuring Pods.
+  Configuration mechanisms within Kubernetes.
```

Comment:
A suggestion from my previous comment on keeping the description about configuring Pods
```suggestion
  Pod configuration mechanisms within Kubernetes.
```